app/javascript/packs/reducers/answers.js

```diff
@@ -56,11 +56,14 @@ function answerFetchFailure(state, action) {
 export function answers(state = {}, action) {
   if (!action) return state;
   switch (action.type) {
-    case ANSWER_FETCH_IN_PROGRESS: return beforeFetchAnswer(state, action);
-    case ANSWER_FETCH_SUCCESS: return answerFetchSuccess(state, action);
-    case ANSWER_FETCH_FAILURE: return answerFetchFailure(state, action);
+    case ANSWER_FETCH_IN_PROGRESS:
```

Comment:
How come there are so many lint changes in your PR? these are going to all show up as your lines of code if anyone runs git blame to figure out who to ask for how the code was written